pom.xml

```diff
@@ -27,7 +27,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${basedir}/ramls/raml-util</ramlfiles_util_path>
-    <vertx.version>3.8.4</vertx.version>
+    <vertx.version>3.9.0</vertx.version>
```

Comment:
The latest 3.9.x version is 3.9.1 containing several bugfixes: https://github.com/vert-x3/wiki/wiki/3.9.1-Release-Notes